gin/sentrygin.go

```diff
@@ -96,3 +182,12 @@ func GetHubFromContext(ctx *gin.Context) *sentry.Hub {
 	}
 	return nil
 }
+
+// StartSpanFromGinContext start a new *sentry.Span from gin.Context.
+func StartSpanFromGinContext(ctx *gin.Context, op string) *sentry.Span {
```

Comment:
We shouldn't expose more ways to start a span. If `StartSpan` can't handle this use-case, we should fix it.

Reply:
For now, in `sentrygin` package, there is no way to access this struct:

https://github.com/getsentry/sentry-go/blob/27e50ad6c8d6c892e2f968ca21061431793bb1ee/tracing.go#L841-L842

Any idea?